conf/tariffLinks.conf

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 HM Revenue & Customs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+urls {
+  tariff {
+    common {
+      dispatchLocation = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-1-message-information-including-procedure-codes#de-11-declaration-type-box-1-declaration-first-subdivision"
+    }
+
+    clearance {
+      dispatchLocation = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-1-message-information-including-procedure-codes#de-11-declaration-type-box-1-declaration-first-subdivision"
+    }
+  }
+}
```

Comment:
How about switching the ordering to have structure like: `urls.tariff.<page>.<journey>`? This way you'd have links for given page closer to each other and it should be easier to update/change them.
I'd rather expect changes to be "change tariff links for page XYZ", instead of "change tariff links for journey XYZ".